tools/build.py

```diff
@@ -60,7 +63,7 @@ def devhelp(help):
     parser.add_argument('--linker-flag', metavar='OPT', action='append', default=[], help='add custom linker flag')
     parser.add_argument('--lto', metavar='X', choices=['ON', 'OFF'], default='ON', type=str.upper, help='enable link-time optimizations (%(choices)s; default: %(default)s)')
     parser.add_argument('--port-dir', metavar='DIR', action='store', default=DEFAULT_PORT_DIR, help='add port directory (default: %(default)s)')
-    parser.add_argument('--profile', metavar='PROFILE', choices=['es5.1', 'minimal', 'es2015-subset'], default='es5.1', type=str.lower, help='specify the profile (%(choices)s; default: %(default)s)')
+    parser.add_argument('--profile', metavar='FILE', action='store', default=DEFAULT_PROFILE, help='specify profile file (default: %(default)s)')
```

Comment:
If the directory of profiles is hard coded, then we can set the 'choices' attribute like before.